### ISSUE TITLE
âncoras não funcionam

### ISSUE THREAD

**Issue by bwstefano** [ecodigital/violencia-indigena#30]
Os links de âncoras levam somente para a página de destino, sem apontar para o local da âncora

**Comment by miguelpeixe** [comment]
em que página você está tentando colocar âncora? o fato das urls serem no padrão `#!` dificultam o uso de âncoras. preciso entender o uso para estudar a solução

**Comment by bwstefano** [comment]
A ideia é linkar diretamente aqueles alertas na página de Dados Abertos (“Existe uma nota metodológica para estes dados”) para o local da página “Metodologia e fontes” em que está a nota.

**Comment by miguelpeixe** [comment]
você pode implementar normalmente, desde que haja um ID de referência para a âncora na página de destino, por exemplo:

http://violenciaindigena.infoamazonia.org/#!/p/1000/#ancora-aqui

**Comment by bwstefano** [comment]
Sim, sim. Mas não tá rolando. Faz o teste, clicando em qualquer alerta desta página: http://violenciaindigena.infoamazonia.org/#!/p/942/ http://violenciaindigena.infoamazonia.org/#!/p/942/

**Comment by miguelpeixe** [comment]
estranho, to vendo aqui o que pode ser
